src/main/resources/mainnet.conf

```diff
@@ -33,9 +33,17 @@ scorex {
       "165.227.26.175:9030",
       "78.46.93.239:9030",
       "159.89.116.15:9030",
-      "83.212.114.255:9030"
+      "83.212.114.255:9030",
+      "136.244.110.145:9030",
+      "91.199.118.161:9030",
+      "104.186.109.55:9030",
+      "209.217.206.254:9030",
+      "94.130.108.35:9030"
     ]
-    maxDeliveryChecks = 20
+
+    # Max number of delivery checks. Stop expecting modifier (and penalize peer) if it was not delivered after that
+    # number of delivery attempts
+    maxDeliveryChecks = 4
```

Comment:
What is the reason to reduce this, is it network throughput problem?
During profiling I observed that network is mostly idle even during sync, as well as the node is 50% idle.
So I don't think we should reduce connectivity parameters, before we observe the actual overhead.
Or, if you have some measurments on this, please share.

Reply:
Syncing performance should be better with maxDeliveryChecks being reduced, as you're waiting less in case of non-delivering peers. 
50% idle time will be fixed with sync protocol 2.0 ( #1226 )